elfi/examples/arch.py

```diff
@@ -0,0 +1,210 @@
+"""Example implementation of the ARCH(1) model."""
+
+import logging
+from itertools import combinations
+
+import numpy as np
+
+import elfi
+
+logger = logging.getLogger(__name__)
+
+
+def get_model(n_obs=100, true_params=None, seed_obs=None, n_lags=5, noise=None):
+    """Return a complete ARCH(1) model.
+
+    Parameters
+    ----------
+    n_obs: int
+        Observation length of the ARCH(1) process.
+    true_params: list, optinal
+        Parameters with which the observed data are generated.
+    seed_obs: int, optional
+        Seed for the observed data generation.
+    n_lags: int, optional
+        Number of lags in summary statistics.
+    noise: int, optional
```

Comment:
I don't think we need the additional `noise`-statistics in the general `arch`-model.